chsdi/templates/htmlpopup/pronatura.mako

```diff
@@ -1,18 +1,18 @@
 <%inherit file="base.mako"/>
 
 <%def name="table_body(c, lang)">
-    <tr><td class="cell-left">${_('ch.pronatura.waldreservate.objnummer')}</td>                         <td>${c['attributes']['objnummer'] or '-'}</td></tr>
-    <tr><td class="cell-left">${_('ch.pronatura.waldreservate.name')}</td>                         <td>${c['attributes']['name'] or '-'}</td></tr>
```

Comment:
name is the only Declaration that is being translated to rm (NOme)... which is not consistent